text/trace/0235-sampling-threshold-in-trace-state.md

```diff
@@ -0,0 +1,87 @@
+# Sampling Threshold Propagation in TraceState
+
+## Motivation
+
+Sampling can theoretically take place at nearly any point in a distributed tracing system. If sampling is to be performed at multiple points in the process, the only way to reason about it effectively is to make sure that the sampling decisions are **consistent**. In this context consistency is the property that allows different points in the sampling chain to make the same sampling decisions, based on information included with the trace.
```

Comment:
> In this context consistency is the property that allows different points in the sampling chain to make the same sampling decisions, based on information included with the trace.

This definition does not match the proposal, because in the proposal nodes are allowed to reduce the threshold-value, thus the sampling decision is not "the same".

Reply:
I agree with @yurishkuro.

Maybe:

In this context, consistency means that a positive sampling decision made at a particular point with probability p1 implies a positive sampling decision made at another point that samples a different piece of information from the same trace with probability p2 >= p1.

Reply:
Thank you for the suggested wording!